builder/FindGlobals.cmake

```diff
@@ -36,25 +36,26 @@ if( Linux OR Darwin )
 
   include( GNUInstallDirs )
 
-  add_definitions(-DUNIX)
+  append( "-DUNIX" CMAKE_FLAGS )
```

Comment:
I have much simpler solution: just deleted the line:). It is not actually used anywhere in sources:
```
[dvrogozh@dvrscl msdk]$ fgrep -rsn UNIX
_studio/shared/umc/core/vm_plus/include/sys/mfx_vm++_pthread_unix.h:21:#ifndef __MFX_VMPLUSPLUS_PTHREAD_UNIX_H__
_studio/shared/umc/core/vm_plus/include/sys/mfx_vm++_pthread_unix.h:22:#define __MFX_VMPLUSPLUS_PTHREAD_UNIX_H__
_studio/shared/umc/core/vm_plus/include/sys/mfx_vm++_pthread_unix.h:33:#endif //__MFX_VMPLUSPLUS_PTHREAD_UNIX_H__
api/mfx_dispatch/linux/CMakeLists.txt:59:add_definitions(-DUNIX)
builder/FindGlobals.cmake:39:  add_definitions(-DUNIX)
builder/FindGlobals.cmake:190:elseif(UNIX)
builder/FindGlobals.cmake:213:if(NOT ERROR AND UNIX)
builder/FindGlobals.cmake:229:if(UNIX AND MFX_GLIBC)
contrib/googletest/googlemock/test/gmock_output_test.py:58:  """Changes all Windows/Mac line endings in s to UNIX line endings."""
contrib/googletest/googletest/include/gtest/gtest.h:1280:  // UNIX epoch.
contrib/googletest/googletest/include/gtest/internal/gtest-port.h:232://                    Extended Regular Expression syntax on UNIX-like platforms
contrib/googletest/googletest/src/gtest-internal-inl.h:566:  // UNIX epoch.
contrib/googletest/googletest/src/gtest-internal-inl.h:904:  // UNIX epoch.
contrib/googletest/googletest/src/gtest.cc:4568:// UNIX epoch.
contrib/googletest/googletest/test/googletest-output-test.py:86:  """Changes all Windows/Mac line endings in s to UNIX line endings."""
[dvrogozh@dvrscl msdk]$
```

And even if it were used, we should have used compiler standard identification instead I think like `__linux__`.

Reply:
done